js/plugins/google-genai/src/googleai/index.ts

```diff
@@ -54,9 +57,13 @@ async function resolver(
 ) {
   switch (actionType) {
     case 'model':
-      if (imagen.isImagenModelName(actionName)) {
+    case 'background-model':
```

Comment:
I know we sometimes lookup `model/veo-`, but that should be fine since veo will define itself as a background-model and then the next lookup for 'background-model/veo' will succeed without resolution.

Reply:
Good point. Thanks.